odoo/addons/base/models/res_partner.py

```diff
@@ -47,7 +47,7 @@ class FormatAddressMixin(models.AbstractModel):
 
     def _fields_view_get_address(self, arch):
         # consider the country of the user, not the country of the partner we want to display
-        address_view_id = self.env.company.country_id.address_view_id
+        address_view_id = self.env.company.country_id.address_view_id.sudo()
```

Comment:
@xmo-odoo @mart-e Do you mind us doing this change.  Seems straightforward, but maybe we can do something better here.

Reply:
This is due to context polution that makes a rebrowse in [`self.env.company`](https://github.com/odoo/odoo/blob/4827b0add7ff62711553f9639ed88c26788fa3d9/odoo/api.py#L561) while it's [already with sudo](https://github.com/odoo/odoo/blob/4827b0add7ff62711553f9639ed88c26788fa3d9/odoo/api.py#L562) without context. Maybe that's a bug in api.py?

Reply:
What is supposed to be already with sudo? `self.env.company`? :fearful:

Reply:
yes, `self.env.user` is [retrieved with the `su` flag](https://github.com/odoo/odoo/blob/06d2b067e56ef7fde5e5ad21e99d868ea5914adb/odoo/api.py#L532), so as is `self.env.user.company_id`. It looks like a bug that `self.env.company` is not always sudo (should either be always, either never)